bin/agent_runner.rb

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+
+# This process is used to maintain Huginn's upkeep behavior, automatically running scheduled Agents and
+# periodically propagating and expiring Events. It also running TwitterStreamAgents and Agents that support long running
+# background jobs.
+
+Dotenv.load if Rails.env == 'development'
```

Comment:
Is this so you don't have to use `foreman`?

`Rails.env.development?` works too